Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -254,22 +253,57 @@ ViewTimeline::Data ViewTimeline::computeViewTimelineData() const
     auto insetStart = m_insets.start.value_or(Length());
     auto insetEnd = m_insets.end.value_or(insetStart);
 
-    auto coverRangeStart = subjectOffset - scrollContainerSize + insetEnd.value();
-    auto coverRangeEnd = subjectOffset + subjectSize - insetStart.value();
+    auto computeRangeStart = [&](SingleTimelineRange range) -> float {
```

Comment:
Is the explicit return type required here?

Reply:
It lets us avoid needing to cast the zero to a float.

Reply:
Couldn't just use `0.f`?

Reply:
Sure.